### ISSUE TITLE
Shader conversion improvement

### ISSUE THREAD

**Issue by artdeell** [ptitSeb/gl4es#163]
Can you improve the shader conversion? Every single shader for Minecraft gives lotta errors when compilation.

**Comment by ptitSeb** [comment]
what kind of error do you have ?
Also, what version of Minecraft, because it used to works fine without errors, and on what hardware are you running?

**Comment by artdeell** [comment]
> what kind of error do you have ?
> Also, what version of Minecraft, because it used to works fine without errors, and on what hardware are you running?

Hardware: MSM8996 Pro Ardeno 530
Minecraft: OptiFine 1.11.2 with Shaders

When joining the world, gl4es fires a lot of `Invalid enum` errors.
Before that errors compiler throws a lot of errors in shaders.

**Comment by ptitSeb** [comment]
Are you sure OptiFine shaders are for OpenGL 2.x and not 3.x?

**Comment by artdeell** [comment]
Yes.

**Comment by artdeell** [comment]
I tried the Lite version of Chocapic13 v7 shaders

**Comment by ptitSeb** [comment]
I took a look. While this is indeed OpenGL 2.x, I see it use the `#include ....` statement. gl4es doesn't support that for now, so I guess this could be the cause of the issue.

**Comment by artdeell** [comment]
Take a look to logcat
[Log_2020-04-23_13-25-30.txt](https://github.com/ptitSeb/gl4es/files/4521820/Log_2020-04-23_13-25-30.txt)

**Comment by ptitSeb** [comment]
is the log coming from Chocapic13v7 Lite shaders? Because I see the errors and the kind it can be, but I don't see were it could from for the shaders sources.

**Comment by artdeell** [comment]
> is the log coming from Chocapic13v7 Lite shaders? Because I see the errors and the kind it can be, but I don't see were it could from for the shaders sources.

No, it's from SFLP shaders (which is opengl 2.0-compartible shaders)

**Comment by ptitSeb** [comment]
Well, I'll see what I can do, but don't hold your breath on this.
